### PR TITLE
Mouseover popUp to show tract info 

### LINKED ISSUE #35: Add onHover popup to census tract map with additional information
At the moment, the user cannot see any information about the census tract that they're hovering over. This task would be to add a pop-up that displays additional information such as the name and number of the census tract.

### PR DESCRIPTION
**Pull request by kmal808**
#### Adding a mouseover popup that shows the census tractname, name20, and pop20

https://user-images.githubusercontent.com/20919083/229788822-608b9b48-9812-46e3-8fc1-b35bf56b709d.mov

Closes #35 (I think, maybe, hopefully)

### CODE REVIEW

File: src/components/censusmap.tsx [hunk 1]
```diff
@@ -57,9 +57,19 @@ const CensusTractMap: NextPage = () => {
...
+          fillOpacity: 0.5,
+        })
+        .bindPopup(
+          "Name: " +
+            feature.properties["tractname"] +
+            " <br> Tract: " +
+            feature.properties["name20"] +
+            " <br> Population: " +
+            feature.properties["pop20"]
+        )
```
Comment (avenmia) [inline src/components/censusmap.tsx:71]: const featureProperties = feature.properties as FeatureProperties;
      // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access, @typescript-eslint/no-unsafe-call
      e.target
        .setStyle({
          fillOpacity: 0.5,
        })
        .bindPopup(
          `Name: ${featureProperties["tractname"]} <br> Tract: ${featureProperties["name20"]} <br> Population: ${featureProperties["pop20"]}}`
        )

**Review by avenmia — CHANGES_REQUESTED** [review]
The build is breaking due to type issues. To fix, add the following interface to the top of the file:
```
interface FeatureProperties {
  tractname: string;
  name20: string;
  pop20: number;
}
```
Then commit the suggestion I made at line 60

File: src/components/censusmap.tsx (hunk 1 above)
Comment (kmal808) [reply src/components/censusmap.tsx]: Template literal done ✅

Comment (kmal808) [reply src/components/censusmap.tsx]: All green on the last run https://github.com/CodeforHawaii/HIERR/actions/runs/4617194461/jobs/8163147255

**Review by avenmia — APPROVED** [review]
lgtm! :shipit:

**Pull request merged**